dpp/src/data_contract/state_transition/data_contract_create_transition/apply_data_contract_create_transition_factory.rs

```diff
@@ -0,0 +1,32 @@
+use crate::{mocks, state_repository::StateRepositoryLike};
+use anyhow::Result;
+
+pub struct ApplyDataContractCreateTransitionFactory<SR>
```

Comment:
The same with the other factories I've mentioned above

Reply:
fixed